### PR TITLE
Ft navbar

### PR DESCRIPTION
**Pull request by mjedwards**
## Description

Created a new Navigation bar, that links to all the proper pages. Also, a new homepage was created.

## Type of change
Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Checklist
Remove any items which are not applicable.

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] My changes generate no new warnings

### CODE REVIEW

File: src/components/DashboardHome/Dashboard.js [hunk 1]
```diff
@@ -15,40 +15,39 @@ import {
...
+  }-${user && user._id}`;
+  const storeLogo = user.imageUrl ? user.imageUrl : NoLogo;
   const copied = () => {
-    message.success('url copied successfully')
-  }
+    message.success("url copied successfully");
+  };
   return (
     <div className='mainDiv'>
+      {/* <Navbar /> */}
```
Comment (ChiragThesia) [inline src/components/DashboardHome/Dashboard.js]: How come this is commented out? Is this not suppose to be here?

File: src/components/DashboardHome/index.js [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
+
 // import Footer from '../DashboardHome/Footer'
 
 const Home = () => {
   return (
     <div>
-      <Dashboard />
-      <Nav />
+      <Navbar />
+      {/* <Dashboard /> */}
```
Comment (ChiragThesia) [inline src/components/DashboardHome/index.js]: Same thing here. if its not being used then you can delete it.

File: src/components/DashboardHome/index.js [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
+      <Orders />
+      {/* <Nav /> */}
```
Comment (ChiragThesia) [inline src/components/DashboardHome/index.js]: here as well.

File: src/components/Orders/Orders.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+import { useDispatch, useSelector } from "react-redux";
+import useCurrency from "../hooks/useCurrency";
+import MonthlySales from "../totoalSales/monthlySales";
+import LifetimeSales from "../totoalSales/lifetimeSales";
+import * as actionCreators from "../../state/actionCreators";
+import { Table, Tag, Button } from "antd";
+import { connect } from "react-redux";
+import "antd/dist/antd.css";
+
+// import { data } from "../../db.js";
```
Comment (ChiragThesia) [inline src/components/Orders/Orders.js]: Please remove the code that is commented out. If its not needed then you can delete it and readd if needed.

**Review by ChiragThesia — APPROVED** [review]
There is some code that is uncommented and still in there. I think just needs a simple clean up otherwise good to go. Good stuff man! Also we may need to update the endpoints at a later time once we get them finalized and working right.

**Review by ChiragThesia — CHANGES_REQUESTED** [review]
Mike, look at the commented section. can you please make the minor changes. Also pull from master once this[ PR](https://github.com/Lambda-School-Labs/shopping-cart-fe/pull/6) is merged to master then do one more push so that URL for test APP is updated. 

Thanks

File: src/less/_home.less [hunk 5]
```diff
@@ -0,0 +1,23 @@
+.order-view {
+  display: flex;
+  flex-flow: column;
+  /* justify-content: space-between; */
```
Comment (erferguson) [inline src/less/_home.less]: remove/delete if not needed

File: src/less/_inventory.less [hunk 6]
```diff
@@ -1,165 +1,162 @@
 .inventory {
-    background-color: @grey;
-    // height: auto;
+  background-color: @grey;
+  // height: auto;
```
Comment (erferguson) [inline src/less/_inventory.less]: remove if not needed

**Review by erferguson — COMMENTED** [review]
small edit updates re: commented out code. great work, well done Mike.

File: src/components/DashboardHome/Dashboard.js (hunk 1 above)
Comment (mjedwards) [reply src/components/DashboardHome/Dashboard.js]: Thank you for your review. All changes have been made.

File: src/less/_home.less (hunk 5 above)
Comment (mjedwards) [reply src/less/_home.less]: Thank you for your review all changes have been made.

**Pull request merged**